xml/System.Windows.Forms/StatusStrip.xml

```diff
@@ -435,9 +435,8 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- Use the <xref:System.Windows.Forms.StatusStrip.LayoutStyle%2A> property to get or set how items are arranged on the <xref:System.Windows.Forms.StatusStrip>.  
-  
-   
+ 
+Use the <xref:System.Windows.Forms.StatusStrip.LayoutStyle%2A> property to get or set how items are arranged on the <xref:System.Windows.Forms.StatusStrip>. Its default value is <xref:System.Windows.Forms.ToolStripLayoutStyle.Table?displayProperty=nameWithType>.
```

Comment:
nit: should you do the same as you did in the next file and style this as inline code instead of a link to itself?